lib/src/cupertino/cupertino_toolbar.dart

```diff
@@ -79,6 +83,14 @@ class CupertinoPopoverToolbar extends StatefulWidget {
   /// All of the items will have this exact height.
   final double height;
 
+  /// The z-coordinate relative to the parent at which to place this toolbar.
```

Comment:
This is a pretty confusing description. Is this how Flutter describes `elevation`?

Reply:
This is the description was extracted from `PhysicalModel`. I added another sentence from `Material`'s description of elevation.